catalog/core/catalog-core-validator/src/test/java/ddf/catalog/validation/impl/MatchAnyValidatorTest.java

```diff
@@ -165,4 +168,22 @@ public void testMatchAnyValidatorPassesWithTwoEnumerationValidators() {
     attributeValidationReportOptional = matchAnyValidator.validate(VALID_ATTRIBUTE_3);
     assertThat(attributeValidationReportOptional.isPresent(), is(false));
   }
+
+  @Test
+  public void testNotEquals() {
+    matchAnyValidator =
+        new MatchAnyValidator(Arrays.asList(enumerationValidator, patternValidator));
+    matchAnyValidator2 =
+        new MatchAnyValidator(Arrays.asList(enumerationValidator2, patternValidator));
+    assertThat(matchAnyValidator.equals(matchAnyValidator2), is(false));
+  }
```

Comment:
❓ Should we also have a test for `Arrays.asList(patternValidator, enumerationValidator)` so the sorting behavior is defined explicitly?

Reply:
[MatchAnyValidatorTest.testEquals()](https://github.com/frnkshin/ddf/blob/20ccdc40dda37c590383f89e3ca434f1e1fc2be6/catalog/core/catalog-core-validator/src/test/java/ddf/catalog/validation/impl/MatchAnyValidatorTest.java#L182) should test for that.